SRTNet.h

```diff
@@ -139,6 +139,18 @@ class SRTNet {
     void
     getActiveClients(const std::function<void(std::map<SRTSOCKET, std::shared_ptr<NetworkConnection>>&)>& function);
 
+    /**
+     *
+     * @brief Get the SRT socket and the network connection context object associated with the connected server. This method
+     * only works when in client mode.
+     * @returns The SRT socket and network connection context of the connected server in case this SRTNet is in client
+     * mode and is connected to a SRT server. Returns {0, nullptr} if not in client mode or if in client mode and not
```

Comment:
No need to distinguish between these two cases? I guess if you do the suggested change of splitting up into two classes later it would become possible automatically.

Reply:
Right now I think it is up to the user of the class to know if it is configured as a server or client and that way know which of these cases it is..